src/Tools/dotnet-monitor/Auth/AuthenticationStartupLogger.cs

```diff
@@ -1,51 +1,42 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Diagnostics.Tools.Monitor.Auth;
 using Microsoft.Extensions.Logging;
+using System;
 
 namespace Microsoft.Diagnostics.Tools.Monitor
 {
     internal sealed class AuthenticationStartupLogger :
```

Comment:
If IAuthHandler.LogStartup is changed as suggested, this becomes greatly simplified. In fact, it can be folded into the existing `AddressListenResultsStartupLogger` class since it is already responsible for logging information based on the `AddressListenResults`.